src/md5.c

```diff
@@ -54,6 +54,10 @@
 #include "md5.h"
 #include <string.h>
 
+/* Really didn't want to have to do this but can't find a better way to do it - SyrianBallaS */
+#include <codeanalysis\warnings.h>
+#pragma warning(disable : ALL_CPPCORECHECK_WARNINGS)
```

Comment:
Didn't want to do this but this is included as a project file, there's no other way around as we don't use a .lib/.dll for it.